lib/foods.js

```diff
@@ -21,7 +53,8 @@ $('document').ready(function() {
   });
 
   $(".food-delete button").on('click', function() {
```

Comment:
be sure to use event delegation here. if there food item is not on the page on page load, this event listener will not be bound